ui/src/components/common/NavHeader.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import { Masthead, MenuItem, Icon } from 'patternfly-react';
-
+import { withRouter } from 'react-router-dom';
 import './NavHeader.css';
 
-const NavHeader = ({ title, user, helpDropdownItems, userDropdownItems }) => (
-  <Masthead title={title} navToggle={false}>
+export const _NavHeader = ({ title, user, helpDropdownItems, userDropdownItems, titleHref, history }) => (
+  <Masthead title={title} navToggle={false} onTitleClick={() => history.push(titleHref)}>
```

Comment:
why not just use the "href" prop for `Masthead`?

Reply:
good catch, I was looking for it and missed it.